modules/acs-architecture-differences-ocp-kube.adoc

```diff
@@ -7,10 +7,6 @@
 
 When you install {product-title-short} on the {ocp}, there are only two architectural differences:
 
-. {product-title-short} installs a lightweight version of Scanner on every secured cluster when you install {product-title-short} on the {ocp} using the Operator or the Helm install method.
-The lightweight Scanner enables the scanning of images in the integrated OpenShift Container Registry (OCR).
-. Sensor communicates with Scanner in the cluster where you have installed Central.
-This connection allows accessing internal registries attached to the cluster.
+. {product-title-short} installs a lightweight version of Scanner on every secured cluster when you install {product-title-short} on {ocp} by using the Operator. The lightweight Scanner enables the scanning of images in the integrated OpenShift Container Registry (OCR). When you install {product-title-short} by using the Helm install method with the _default_ values, the lightweight version of Scanner is not installed. To install the lightweight Scanner on the secured cluster by using Helm, you must set the `scanner.disable=false` parameter.
 
-.{product-title} architecture for {ocp}
-image::acs-architecture-ocp.png[{product-title} architecture for {ocp}]
+. Sensor can communicate with the lightweight Scanner installed in the secured cluster. This connection allows Sensor to access registries directly from the secured cluster in scenarios where Central might be unable to access them. Scanner requests updated data from Sensor, Sensor forwards these requests to Central, and Central downloads the requested data from `definitions.stackrox.io`.
```

Comment:
This doesn't really seem like a difference, so I think there may only be one difference between the two (OCP and K8s) now

Reply:
That's a good point, perhaps we should remove the 'architectural differences' section all together.

Reply:
I changed this a bit and moved it to reflect the installation differences (which I think was originally the point of that section, or at least something that we wanted users to know - maybe it just morphed over different revisions). I also removed the part about lightweight scanner communication since it is described (and perhaps more appropriately located) in this section: https://80974--ocpdocs-pr.netlify.app/openshift-acs/latest/architecture/acs-architecture.html#acs-architecture-interaction-between-services_acs-architecture